schemas/questionnaire_v1.json

```diff
@@ -155,6 +155,10 @@
             "$ref": "https://eq.ons.gov.uk/common_definitions.json#/non_empty_string",
             "description": "Title used for the navigation container."
           },
+          "page_title": {
+            "$ref": "https://eq.ons.gov.uk/common_definitions.json#/non_empty_string",
+            "description": "Custom Page Title used for section summary pages."
+          },
```

Comment:
I think this would be better placed in under the `summary` property.

Reply:
I went back and fore on this - happy to go with the above suggestion though.